patches/sdk/0002-Add-missing-PlatformAbstractions-reference.patch

```diff
@@ -1,24 +1,24 @@
-From 2dfcc39cbafe78da5b81979b9caa256bea72abdc Mon Sep 17 00:00:00 2001
+From 21b9c9bd8f623ec769a544b9ac9a6ccf4609ef60 Mon Sep 17 00:00:00 2001
 From: Chris Rummel <crummel@microsoft.com>
-Date: Tue, 30 Apr 2019 22:46:42 -0500
-Subject: [PATCH] Add missing PlatformAbstractions reference.
+Date: Thu, 23 May 2019 01:20:00 -0500
+Subject: [PATCH 2/3] Add missing PlatformAbstractions reference.
 
 ---
  .../Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj   | 1 +
  1 file changed, 1 insertion(+)
 
 diff --git a/src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj b/src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj
-index d437e1d1..63b2f67b 100644
+index 41bbd113..848504be 100644
 --- a/src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj
 +++ b/src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj
-@@ -44,6 +44,7 @@
+@@ -45,6 +45,7 @@
      <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkVersion)" ExcludeAssets="Runtime" />
      <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCoreVersion)" ExcludeAssets="Runtime" />
      <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="$(MicrosoftExtensionsDependencyModelVersion)" />
 +    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="$(MicrosoftDotNetPlatformAbstractionsPackageVersion)" />
+     <PackageReference Include="Microsoft.NET.HostModel" Version="$(MicrosoftNETHostModelVersion)" />
```

Comment:
Will this add a new prebuilt?

Reply:
Yes, but this is existing code being moved (see https://github.com/dotnet/core-setup/issues/6338).